lib/v2/panda/index.js

```diff
@@ -0,0 +1,29 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const feedId = ctx.params.id;
+
+    const rootUrl = 'https://api-panda.com/v4/';
+    const apiUrl = `${rootUrl}articles?feeds=${feedId}&limit=30&page=1&sort=popular`;
```

Comment:
You can use `ctx.query.limit` which allows user to specify the limit they want instead of hard coding it